katas/dollars_and_cents.js

```diff
@@ -0,0 +1,5 @@
+//from https://www.codewars.com/kata/55902c5eaa8069a5b4000083
+
+let formatMoney =
+  amount =>
+    "$"+amount.toFixed(2) 
```

Comment:
You could make use of ES6 template strings, and place parameters on same line.  Given the concise nature of this function, I would probably write it on a single line:
```js
const formatMoney = amount => `$${amount.toFixed( 2 )}`
```